core/modules/contact/contact.install

```diff
@@ -78,6 +78,17 @@ function contact_update_1001() {
   }
 }
 
+/**
+ * Set default value for new 'Phone' field setting.
+ */
+function contact_update_1002() {
```

Comment:
OK, here's the update hook, but where is the change to the default contact.settings.json?

Reply:
Ha, good point!